lib/index.js

```diff
@@ -16,15 +16,22 @@ const action = {
         const comment = github.context.payload.comment.body;
         const args = String(comment).trim().split(/[\r\n ]+/);
 
-        core.info(`Using config: ${JSON.stringify(config, undefined, 2)}`);
-        core.info(`Parsing comment: ${comment}`);
+        core.info(`Validating config: ${JSON.stringify(config, undefined, 2)}`);
+
+        try {
+            this.validateConfig(config);
+        } catch (error) {
+            return this.fail(error.message);
+        }
 
         // Add in the implied forward slash for the first layer of commands. This makes
         // it easier to parse and format error messages later.
         for (let command of config.commands) {
-            command.name = `/${command.name}`;
+            command.prefix = '/';
```

Comment:
Directly mutating the user's input this way has the side-effect of implicitly supporting an arbitrary "prefix" property from the configuration file, with undefined behavior.

Reply:
I think you could avoid this side-effect, and also simplify a few cases, by removing the slash from the first argument instead.